included/README.md

```diff
@@ -93,7 +93,9 @@
  
 [cypress/included:9.5.4](9.5.4) | `cypress/browsers:node16.14.0-slim-chrome99-ff97`
  
-[cypress/included:9.6.0](9.6.0) | `cypress/browsers:node16.14.2-slim-chrome100-ff99-edge` 
+[cypress/included:9.6.0](9.6.0) | `cypress/browsers:node16.14.2-slim-chrome100-ff99-edge`
+ 
+[cypress/included:9.6.1](9.6.1) | `cypress/browsers:node16.14.2-slim-chrome100-ff99-edge` 
```

Comment:
@jordanpowell88 we need to update the script that auto-genertates these markdown updates. The extra space in these files breaks the markdown table.

Reply:
We are looking into it @emilyrohrbough . Thanks